src/external/views/nunjucks/view-licences/purposes.njk

```diff
@@ -0,0 +1,113 @@
+{% extends "./nunjucks/layout.njk" %}
+{% from "nunjucks/macros/point.njk" import point %}
+{% from "summary-list/macro.njk" import govukSummaryList %}
+
+{% block content %}
+
+  {% if crmData.document_name%}
+    {% set licenceName %} Abstraction points for {{crmData.document_name}}{% endset %}
+    {{ title(licenceName, pageTitle) }}
+  {% else %}
+    {{ title(pageTitle) }}
+  {% endif %}
+
+  {% for purpose in licenceData.purposes %}
+    <h3 class="govuk-heading-m">{{ purpose.name }}</h3>
+    <hr class="govuk-section-break govuk-section-break--visible">
+
+    <dl class="govuk-summary-list">
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key govuk-!-font-weight-regular">
+        {{ 'Abstraction point' | pluralize(purpose.points) }}
+      </dt>
+
+        {% if purpose.points.length == 1 %}
+        <dd class="govuk-summary-list__value">
+          {{ purpose.points[0] | ngrPoints() }}
+        {% elif purpose.points.length < 6 %}
+        <dd class="summary-list__value--nested-list">
+          <dl class="summary-list--nested-list">
+            {% for abstractionPoint in purpose.points %}
+                <div class="govuk-summary-list__row">
+                  <dd class="{% if loop.index == purpose.points.length %} summary-list__value--no-bottom-border {%else%} govuk-summary-list__value  {%endif%}">
+                    {{ point(abstractionPoint) }}
+                  </dd>
+                </div>
+            {% endfor %}
+          </dl>
+        {% else %}
+        <dd class="summary-list__value--nested-list">
+          <details>
+            <summary>View your {{ purpose.points.length }} abstraction points for {{ purpose.name | lower }}</summary>
+            <dl class="govuk-summary-list summary-list--nested-list">
+              {% for abstractionPoint in purpose.points %}
+                  <div class="govuk-summary-list__row">
+                    <dd class="{% if loop.index == purpose.points.length %} summary-list__value--no-bottom-border {%else%} govuk-summary-list__value  {%endif%}">
+                      {{ point(abstractionPoint) }}
+                    </dd>
+                  </div>
+              {% endfor %}
+            </dl>
+          </details>
+        {% endif %}
+      </dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key govuk-!-font-weight-regular">
+        Period of abstraction
+      </dt>
+      <dd class="govuk-summary-list__value">
+        From {{ purpose.periodStart | abstractionPeriod() }}
```

Comment:
If the filter has no arguments, I think the brackets may be optional, e.g.:
`{{ purpose.periodStart | abstractionPeriod }}`